src/app/pages/Search/Search.module.css

```diff
@@ -0,0 +1,30 @@
+.container {
+  display: grid;
+  grid-template-rows: auto 1fr;
+  max-width: 500px;
```

Comment:
Do you have this `max-width` on every page? If so, add it to the `App.tsx`.

Reply:
Yes I have that everywhere. But in the end it will be replaced by a mobile layout.

Reply:
So in the end you have to remove it from every page instead of App.tsx only ;)